### PR TITLE
feat(user): add component & doc

### PR DESCRIPTION
**Pull request by Lzumikonata**
## PR Checklist

- [x] Fix linting errors
- [x] Label has been added

### CODE REVIEW

File: docs/zh-cn/components/user.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+展示用户信息。
+
+</ex-code>
+
+<ex-code name="ex-user-description"></ex-code>
+
+<ex-code name="ex-user-social"></ex-code>
+
+<ex-footer edit-link="https://github.com/zeit-ui/vue/edit/master/docs/en-us/components/user.md">
```
Comment (unix) [inline docs/zh-cn/components/user.md:14]:
```suggestion
<ex-footer edit-link="https://github.com/zeit-ui/vue/edit/master/docs/zh-cn/components/user.md">
```

File: docs/zh-cn/components/user.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+| Attribute | Description | Type | Accepted values | Default
```
Comment (unix) [inline docs/zh-cn/components/user.md:16]: translation required.

File: packages/user/user.vue [hunk 3]
```diff
@@ -0,0 +1,24 @@
+<template lang="pug">
+.zi-user(v-on="$listeners")
+  zi-avatar.zi-user-avatar(:src="src" :text="text" v-bind="$attrs")
```
Comment (unix) [inline packages/user/user.vue:3]: Import `Avatar` component manually.

**Comment by msurguy** [comment]
Thank you, @unix  and @Lzumikonata !

**Pull request merged**